ayeletK/assignment3/position.cpp

```diff
@@ -0,0 +1,19 @@
+class Position{
+public:
+    int _row;
+    int _col;
+    Position(): Position(-1 ,-1) {}
+    Position(int row, int col): _row(row), _col(col) {}
+    Position(const Position& pos): _row(pos._row), _col(pos._col) {}
+    Position& operator=(const Position& pos) {
+        _row = pos._row;
+        _col = pos._col;
+        return *this;
+    }
+    bool operator<(const Position& pos) const{
+        return _row < pos._row || _col < pos._col;
+    }
+    bool equal(Position pos){
```

Comment:
You can overload `==` operator:
```C++
bool operator==(Position pos) {
    return _row == pos._row && _col == pos._col;
}
```
And then compare using the operator directly:
```C++
Position p1(1,2), p2(3,4);
return p1 == p2;
```

Unless there is a specific reason to have a function (for example. want to keep a function pointer or use it in other contexts), it is common to use this power of C++ and override the existing compare operator.

Reply:
Ok! I'll fix it.